test.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
```

Comment:
Delete this.  Test files should not go to production.

Reply:
Remove this when you are done testing.  It does not belong in published/released code.